docs/api.md

```diff
@@ -406,7 +406,7 @@ Vantiq Document. Providing both `filename` and `doc_name` in this case is an err
 
 ### Examples
 ```python
-vr: VantiqResponse = await client.upload(VantiqResources.DOCUMENTS, 'iamge/png', '/file/name.png')
+vr: VantiqResponse = await client.upload(VantiqResources.DOCUMENTS, 'image/png', 'file/name.png')
 if vr.is_success:
     # Here, we will have uploaded a file named `/file/name.png`,
```

Comment:
should change here, too (or I can just add all of these to the count PR with the insert/update/upsert stuff, etc.